ext/storage_sources/local_store/local_store.c

```diff
@@ -485,10 +502,17 @@ local_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *sessi
         ret = local_err(local, session, ENOMEM, "local_file_system.auth_token");
         goto err;
     }
+
+    /*
+     * The home directory owned by the connection is valid for as long as the connection is open,
+     * which is longer than this file system will be open.
```

Comment:
While what you say is true I'm not fully getting why you're saying it. Are you saying "it is okay to retain the home directory because it cannot change over the lifetime of the connection"? Or "the memory address will remain valid because..."?

Reply:
Yes, both of those are true.  I've updated the comment, hopefully that will clarify.